examples/federated-learning/tff/federated-learning-for-image-classification/workload-pkg/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  labels:
+    app: fl-image-classification
+  name: fl-image-classification
+  namespace: ns  # kpt-set: ${namespace}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: fl-image-classification
+  template:
+    metadata:
+      labels:
+        app: fl-image-classification
+    spec:
+      containers:
+        - name: fl-image-classification
+          image: fl-image-classification:latest  # kpt-set: ${tff-runtime-container-image-id}
+          ports:
+            - containerPort: 8888
```

Comment:
Do we need a corresponding `Service` for this `Deployment`?

Reply:
Done